src/Nox.Reference.Data.Common/Seeds/NoxReferenceDataSeederBase.cs

```diff
@@ -68,6 +68,6 @@ protected virtual void DoSpecialTreatAfterAdding(IEnumerable<TSource> sources, I
         {
         }
 
-        protected abstract IEnumerable<TSource> GetDataInfos();
+        protected abstract List<TSource> GetDataInfos();
```

Comment:
Why list?

Reply:
Because list has bigger interface, and we are using lists most everywhere anyway
So way not to utilize list we are using anyway?

Reply:
My point of view on whether to use List or IEnumerable is to consider method as result of immutable units.
Like you mentioned List has bigger interface however it'd better get simple interface as possible so that we couldn't manipulate with set;
var list = GetItems();
list .Add(item);
// rather
var list = GetItems();
var t = new List<>(list); // if we want to manipulate retrieved data.

Reply:
I don't see a big importance of it here, but in that case we can use IReadOnlyList instead, and not just completely remove all list functionality by hiding in under IEnumerable
IReadOnlyList implements IEnumerable and has this functionality in it, right?